editor.planx.uk/src/routes/views/authenticated.tsx

```diff
@@ -16,6 +16,8 @@ export const authenticatedView = async () => {
 
   await useStore.getState().initUserStore(jwt);
 
+  useStore.getState().setPreviewEnvironment("editor");
```

Comment:
I'm still keen to tidy this up at some point or to find an alternative way of tracking the high-level state of the application across components and routes. 

Some thoughts from a while back here -  https://trello.com/c/0Uw8ViZJ/2324-make-editor-sidebar-preview-behave-same-as-the-unpublished-preview-increase-granularity-of-previewenvironment